tests/tcl/tests/support/server.tcl

```diff
@@ -403,7 +403,6 @@ proc start_server {options {code undefined}} {
 
     set unixsocket [file normalize [format "%s/%s" [dict get $config "dir"] "socket"]]
     dict set config "unixsocket" $unixsocket
-    dict set config "log-dir" [format "%s/%s" [dict get $config "dir"] "stdout"]
```

Comment:
We can redirect log to stdout by setting "log-dir" to "stdout", but we should later update all reference hardcode to `$dir/kvrocks.INFO`